include/toBeSorted/tlist.h

```diff
@@ -8,122 +8,126 @@
 // * Swapped node and count members
 // Used in DowsingTarget, maybe also in dAcBase_c.soundStuff
 
-// It's not immediately clear whether this stores pointers to nodes
-// or pointers to objects. DowsingTarget's node offset is 0, so there's
-// no observable difference, but the SoundSource stuff makes it
-// look like it stores object pointers
+#define TLIST_NODE_DEF(class) TListNode<class> mLink
 
+#define TLIST_LIST_DEF(class, name) TList<class, offsetof(class, mLink)> name
```

Comment:
Let's just get rid of both these macros - the first one hardcodes the name and the second one isn't used.